tests/unit/test_lua.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+
+from lupa._lupa import LuaRuntime
+from dredis.keyspace import Keyspace
+from dredis.lua import RedisScriptError
+from dredis.lua import LuaRunner, RedisLua
+
+
+def test_lua_return_redis_types_run():
+    k = Keyspace()
+    runner = LuaRunner(k)
+    lua_script = """return {'test', true, false, 10, 20.3, {4}}"""
+
+    assert runner.run(lua_script, [], []) == ['test', 1, None, 10, 20, [4]]
+
+
+def test_lua_table_with_error_run():
+    k = Keyspace()
+    runner = LuaRunner(k)
+    lua_script_err = """return {err='This is a ValueError'}"""
+
+    with pytest.raises(ValueError) as e:
+        runner.run(lua_script_err, [], [])
+
+    assert str(e.value) == 'This is a ValueError'
+
+
+def test_lua_table_with_ok_run():
+    k = Keyspace()
+    runner = LuaRunner(k)
+    lua_script_ok = """return {ok='Everything is OK'}"""
+
+    assert runner.run(lua_script_ok, [], []) == 'Everything is OK'
+
+
+def test_redislua_return_lua_types_call():
+    k = Keyspace()
+    lua_runtime = LuaRuntime(unpack_returned_tuples=True)
+    redis_lua = RedisLua(k, lua_runtime)
+    lua_script = """return {'test', true, false, 10, 20.3, {'another string'}}"""
+    table = redis_lua.call('EVAL', lua_script, 0, [])
+
+    assert table[1]['ok'] == 'test'
```

Comment:
I think this is incorrect. Regular strings should be just the string, not a table.

Using real redis:

Strings
```text
127.0.0.1:6379> set foo "bar"
OK
127.0.0.1:6379> eval "return redis.call('get', 'foo')['ok']" 0
(nil)
127.0.0.1:6379> eval "return redis.call('get', 'foo')" 0
"bar"
```

Simple String
```text
127.0.0.1:6379> eval "return redis.call('ping')['ok']" 0
"PONG"
```

Reply:
@hltbra I see, I am changing back to what we had and updating the tests to reflect this.